### DISCUSSION TITLE
Keyboard navigation for a faster experience

### DISCUSSION THREAD

**Discussion by jasiralavi** [dailydotdev/daily#1415] [Feature Requests]
It would be super cool to have full navigation control using keyboard. I'm surprised devs here have not prioritized this, this being a platform for devs.

- Use arrow keys to navigate My Feed. Use tab after that to srcoll though the buttons after that (already works)
- Space or Enter to open the post (works now with enter)
- Shortcuts to Bookmark, save, etc from the feed as well as from when the news item is open. For example: 
→ Tab+B to Bookmark, 
→ Tab+C to copy link, 
→ Tab+U or T++ to Upvote,
→ Tab+D or Tab+- to Downvote,
→ Tab+K for search,
→ Tab+N for New Post,
→ Tab+M to show other menu items (the actions list),  
→ Tab+T to toggle navigation focus between the feed and the left sidebar, 
→ Tab+H to open Help and shortcuts cheat-sheet, etc.
- It can be Ctrl / Tab / Alt etc. But let users configure it. This is important since different OS and browsers behave different. I use firefox on Linux and have both search-box and address-box  both on, so for me Ctrl+K takes my cursor focus to the browser search-box, not to the dev.daily search

**Comment by AyushmanOfficial** [comment]
How could someone use shortcuts with tab because tab does a navigation for accessibility right? Keyboard Navigation? It Can Be Somewhat like vscode does Ctrl+K & then Ctrl+O  tab could be easily replaced with Alt (Windows, Linux) Option (MacOS)

I use linux too though not arch btw !

**Reply by jasiralavi** [reply]
Tab+ shortcuts are used by many webapps. See Asana for example:
https://help.asana.com/s/article/keyboard-shortcuts?language=en_US

**Reply by AyushmanOfficial** [reply]
Oh no but what about accessiblity?

**Comment by idoshamun** [comment]
Hi @jasiralavi,

Thanks for the suggestion. Let's keep the discussion here: https://github.com/dailydotdev/daily/discussions/1017
